aion/web/static/assets/svg/hemp.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Should we upload this to the repo?

Reply:
Yes, we should keep svg needed for layout in repo.

Reply:
Ack.